src/lib/widget/inline.svelte

```diff
@@ -191,41 +49,43 @@
   // A reference to the `form` DOM element
   let formEl: HTMLFormElement;
 
-  // Set base config to SDK
-  // TODO: Move to a shared utility
-  configure({
-    // Set some basics by default
-    ...{
-      // TODO: Could this be a default OAuth client provided by Platform UI OOTB?
-      clientId: 'WebLoginWidgetClient',
-      // TODO: If a realmPath is not provided, should we call the realm endpoint and detect a likely default?
-      // https://backstage.forgerock.com/docs/am/7/setup-guide/sec-rest-realm-rest.html#rest-api-list-realm
-      realmPath: 'alpha',
-      // TODO: Once we move to SSR, this default should be more intelligent
-      redirectUri:
-        typeof window === 'object' ? window.location.href : 'https://localhost:3000/callback',
-      scope: 'openid email',
-    },
-    // Let user provided config override defaults
-    ...config,
-    // Force 'legacy' to remove confusion
-    ...{ support: 'legacy' },
-  });
+  if (config) {
+    // Set base config to SDK
+    // TODO: Move to a shared utility
+    configure({
+      // Set some basics by default
+      ...{
+        // TODO: Could this be a default OAuth client provided by Platform UI OOTB?
+        clientId: 'WebLoginWidgetClient',
+        // TODO: If a realmPath is not provided, should we call the realm endpoint and detect a likely default?
+        // https://backstage.forgerock.com/docs/am/7/setup-guide/sec-rest-realm-rest.html#rest-api-list-realm
+        realmPath: 'alpha',
+        // TODO: Once we move to SSR, this default should be more intelligent
+        redirectUri:
+          typeof window === 'object' ? window.location.href : 'https://localhost:3000/callback',
+        scope: 'openid email',
+      },
+      // Let user provided config override defaults
+      ...config,
+      // Force 'legacy' to remove confusion
+      ...{ support: 'legacy' },
+    });
+  }
 
   /**
    * Initialize the stores and ensure both variables point to the same reference.
    * Variables with _ are the reactive version of the original variable from above.
    */
-  let _journeyStore = (journeyStore = initializeJourney(config));
-  let _oauthStore = (oauthStore = initializeOauth(config));
-  let _userStore = (userStore = initializeUser(config));
+  api.setJourneyStore(initializeJourney(config));
+  api.setOAuthStore(initializeOauth(config));
+  api.setUserStore(initializeUser(config));
```

Comment:
Now that we are using this `api` object created from the factory, we don't need the double references between the `script` here and the one (`context="module"`) from above. Using an object instead allows for passing the needed stores by reference, rather than value.